### ISSUE TITLE
Batch Normalization After EdgeConv

### ISSUE THREAD

**Issue by gycn** [rusty1s/pytorch_geometric#271]
## ❓ Questions & Help

Couldn't tell if this question has already been answered. I implemented EdgeConv as per the demonstration and I want to apply batch normalization to the output of the convolution operation. However, since the data is concatenated together, I can't use the pytorch batchnorm classes. Is there a batchnorm function implemented for this purpose? If not, what would be the best way of implementing something like this? Thank you.

**Comment by rusty1s** [comment]
Hi, you can use BatchNorm1d when features have shape `[batch_size * num_nodes, num_features]`. We do this, e.g., in the GIN example `mutag_gin`.
